modules/querier/querier.go

```diff
@@ -376,12 +376,12 @@ func forOneIngesterRing(ctx context.Context, replicationSet ring.ReplicationSet,
 
 		client, err := pool.GetClientFor(ingester.Addr)
 		if err != nil {
-			return nil, errors.Wrap(err, fmt.Sprintf("failed to get client for %s", ingester.Addr))
+			return nil, fmt.Errorf(fmt.Sprintf("failed to get client for %s: ", ingester.Addr), "%w", err)
```

Comment:
I think the inner `fmt.Sprintf` isn't strictly necessary
```suggestion
			return nil, fmt.Errorf("failed to get client for %s:  %w", ingester.Addr, err)
```